### DISCUSSION TITLE
MiSTer: Symlink roms

### DISCUSSION THREAD

**Discussion by Toryalai1** [retronas/discussions#12] [Q&A] [answered]
Hello,
I wonder if and how it is possible to symlink roms from one folder to another folder? Here is the deal:
A new "Mega Drive" core is going to be launched for MiSTer which will "live" together with the existing core (which is called genesis). The new core which is in development will be called MegaDrive. How can I symlink the roms from the Genesis folder to a new folder that I will create that is called MegaDrive?

**Comment by sairuk** [comment, marked as answer]
you would need to do it manually for now but it'd be something like this as root

```
cd /data/retronas/mister/games
ln -s ../../roms/sega/megadrive MegaDrive
```
then repeat that for saves, states, bios etc

**Reply by Toryalai1** [reply]
Great! Thank you for the response. 
I
Just one quick off topic question, I've always placed my roms in /mister/games folder. Is this the correct way of doing it or should I place them in retronas/roms/.. folders?

**Reply by sairuk** [reply]
should be fine, they point to the same location in retronas
